pkg/client/client_test.go

```diff
@@ -257,7 +257,7 @@ var _ = Describe("ConmonClient", func() {
 	Describe("ExecSyncContainer", func() {
 		for _, terminal := range []bool{false} {
```

Comment:
Let's re-enable the terminal tests here.

Reply:
oops, good catch, I did so here and https://github.com/containers/conmon-rs/pull/327